corehq/apps/reports/formdetails/readable.py

```diff
@@ -170,6 +171,10 @@ class CaseTypeMeta(JsonObject):
     error = StringProperty()
     has_errors = BooleanProperty()
 
+    # store where this case type gets loaded so that we can look it up more easily later
+    load_properties = DictProperty()  # {"form_id:question_path" -> [CaseProperty, ...]}
```

Comment:
I'm generally opposed to magically-formatted strings, even though this isn't used in many places. Thoughts on changing this to `{form_id -> {question_path -> [CaseProperty, ...]}}`?

Reply:
Yeah, that is much better. I have no idea what I was thinking with that...